.ci/Jenkinsfile

```
@@ -204,13 +207,18 @@ def withKubernetes(Closure body) {
 
 def prepareStack() {
   def stackArgs = '-v'
-  def manifest = readYaml(file: "manifest.yml")
 
-  if (manifest != null && manifest["conditions"] != null) {
-    def kibanaVersionCondition = manifest["conditions"]["kibana.version"]
-    if (kibanaVersionCondition != null) {
-      def version = findOldestSupportedVersion(versionCondition: kibanaVersionCondition)
-      stackArgs += " --version ${version}"
+  if ( env.STACK_VERSION != '' && env.STACK_VERSION != null ) {
```

`?.trim()` is the same:

```suggestion
  if ( env.STACK_VERSION?.trim() ) {
```